extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/pom.xml

```diff
@@ -57,14 +57,35 @@
         <dependency>
             <groupId>org.mybatis</groupId>
             <artifactId>mybatis</artifactId>
-            <version>3.5.9</version>
+            <version>3.5.11</version>
         </dependency>
         
         <!-- MyBatis Guice -->
         <dependency>
             <groupId>org.mybatis</groupId>
             <artifactId>mybatis-guice</artifactId>
-            <version>3.10</version>
+            <version>3.17</version>
+            <exclusions>
+
+                <!-- This dependency appears to be necessary only to provide an
```

Comment:
I understand that this dependency isn't necessary for us, but I'm still unclear why it's being explicitly excluded. Doesn't seem like it could be for license reasons since it's MIT-licensed, and furthermore there's an explicit inclusion in `extensions/pom.xml` for the same artifact.

Reply:
Well, the inclusion in `extensions/pom.xml` was an initial experiment that was inadvertently committed - I'll get rid of that.

It's excluded because we have that build-time sanity check that fails the build if any extension bundles SLF4J binaries (to prevent compatibility issues with the SLF4J bundled with the webapp). Thus, we have to do one of the following:

* Exclude the dependency if it can be verified that it's unused (I went this route)
* Override that enforcer rule within this `pom.xml` to explicitly allow bundling of `jcl-over-slf4j`.

I went the exclusion route as it was easy to verify that the code in question doesn't use commons-logging.

Reply:
Cool, makes sense.

Reply:
Amended the commit to remove the change to `extensions/pom.xml`.